api/services/orders/order_handler.go

```diff
@@ -274,3 +275,24 @@ func isValidStatusUpdate(currentStatus string, newStatus string) error {
 
 	return fmt.Errorf("order status can not change from '%v' to '%v'", currentStatus, newStatus)
 }
+
+// @Summary Retrieve order details by invoice number
+// @Description Retrieves order details and item list by invoice number.
+// @Tags Orders
+// @Produce json
+// @Security Bearer
+// @Param invoice_number path string true "Invoice Number"
+// @Success 200 {object} types.OrderByInvoiceResponse
```

Comment:
This should be an `object` of `types.Order`

Reply:
Fixed!